Sources/Reachability/Reachability.h

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Just one to note, I believe this (Adding Obj-C files) will break your SPM support 😞

Reply:
Left no choice really :( Or use another Reachability.
Using method swizzling in ObjC which I think in swift needs the ObjC runtime anyway :/.
Not too fussed (atm anyway)